### ISSUE TITLE
[Bug]: Kilograms and Count are missing Type Alias challenge

### ISSUE THREAD

**Issue by is-it-ayush** [typehero/typehero#1031]
### What happened?

The following types `Kilograms` and `Count` are missing on [type-aliases challenge](https://typehero.dev/challenge/type-aliases). This should be mentioned in the challenge description under "Solve the challenge" section.

### What browsers are you seeing the problem on?

All

**Comment by dimitropoulos** [comment]
Did you see the section that reads:

```
  // the tests show that you need a `mass` property here
  // but first you might need to create an alias for `Kilograms`
  // because that's the value of `mass`
```

The idea is to give you some practice actually creating an alias from scratch, which for this one challenge means we can't pre-write it for you if the goal is for it to really give you some practice making type aliases.  So, in this case it's intended behavior for you to have to write `type Kilograms = `.  Does that make sense?

**Comment by is-it-ayush** [comment]
I did read that @dimitropoulos! This issue primarily was for `Count` although I included `Kilograms` since it was also only mentioned in the comments. `Count` isn't mentioned anywhere. I think it'd be better to mention them in the challenge description so one doesn't have to search the UI for what to write from a user's perspective who is expected to be new to typescript. This is suggestion so if it isn't something that is planned. We could close this issue!
